src/dmd/mars.d

```diff
@@ -998,6 +970,45 @@ private int tryMain(size_t argc, const(char)** argv)
     return status;
 }
 
+private void generateJson(Modules* modules)
+{
+    OutBuffer buf;
+    json_generate(&buf, modules);
+
+    // Write buf to file
+    const(char)* name = global.params.jsonfilename;
+    if (name && name[0] == '-' && name[1] == 0)
```

Comment:
`strcmp`

Reply:
This is existing code, I think any changes to it would be a seperate PR.